Lombiq.HelpfulLibraries.LinqToDb/LinqToDbConnection.cs

```diff
@@ -13,6 +13,10 @@ public LinqToDbConnection(IDataProvider dataProvider, DbTransaction dbTransactio
         : base(dataProvider, dbTransaction) =>
             _tablePrefix = tablePrefix;
 
+    /// <summary>
+    /// Overrides table name with a prefixed table name, optionally using the provided
```

Comment:
In reference to the "Get" in the method's name, i.e. also include that it returns the table? If so, adding `<returns>The original table but with a prefixed name.</returns>` seems sufficient to me. Otherwise it's not clear to me what else to include in the description regarding getting the table.

Reply:
Check out what the returned type does, and how it's used, to understand it better.